src/methods/getColumn.ts

```diff
@@ -0,0 +1,37 @@
+import BinaryTree from "./binaryTree"
+import { TreeNode } from "./binaryTree";
+
+interface GetColumn extends BinaryTree<number> {
+    getColumn(node: TreeNode<number>, column: number): number[]
+}
+
+let currentColumn: number;
+let getColumnResult: number[];
+
+class GetColumn implements GetColumn {
```

Comment:
The `getColumn` method should be under `BinaryTree` functionality. You don't need a separate class for that. See the [task requirements](https://github.com/evolution-gaming/typescript-bootcamp/tree/main/homeworks/03-data-structure#binary-search-tree-requirements).